options.js

```diff
@@ -1,139 +1,193 @@
-	localStorage['disablekeys'] = document.optform.disablekeys.checked;
-	localStorage['kanjicomponents'] = document.optform.kanjicomponents.checked;
-
-	var kanjiinfoarray = new Array(chrome.extension.getBackgroundPage().rcxDict.prototype.numList.length/2);
-	numList = chrome.extension.getBackgroundPage().rcxDict.prototype.numList;
-	for (i = 0; i*2 < numList.length; i++) {
-		localStorage[numList[i*2]] = document.getElementById(numList[i*2]).checked;
-		kanjiinfoarray[i] = localStorage[numList[i*2]];
-	}
-
-	localStorage['lineEnding'] = document.optform.lineEnding.value;
-	localStorage['copySeparator'] = document.optform.copySeparator.value;
-	localStorage['maxClipCopyEntries'] = document.optform.maxClipCopyEntries.value;
-
-	var popupDelay;
-	try {
-		popupDelay = parseInt(document.optform.popupDelay.value);
-		if (!isFinite(popupDelay)) {
-			throw Error('infinite');
-		}
-		localStorage['popupDelay'] = document.optform.popupDelay.value;
-	} catch (err) {
-		popupDelay = 150;
-		localStorage['popupDelay'] = "150";
-	}
-	localStorage['showOnKey'] = document.optform.showOnKey.value;
-	localStorage['popupLocation'] = document.optform.popupLocation.value;
-
-	chrome.extension.getBackgroundPage().rcxMain.config.css = localStorage["popupcolor"];
-	chrome.extension.getBackgroundPage().rcxMain.config.highlight = localStorage["highlight"];
-	chrome.extension.getBackgroundPage().rcxMain.config.textboxhl = localStorage["textboxhl"];
-	chrome.extension.getBackgroundPage().rcxMain.config.onlyreading = localStorage["onlyreading"];
-	chrome.extension.getBackgroundPage().rcxMain.config.minihelp = localStorage["minihelp"];
-	chrome.extension.getBackgroundPage().rcxMain.config.popupDelay = popupDelay;
-	chrome.extension.getBackgroundPage().rcxMain.config.disablekeys = localStorage["disablekeys"];
-	chrome.extension.getBackgroundPage().rcxMain.config.showOnKey = localStorage["showOnKey"];
-	chrome.extension.getBackgroundPage().rcxMain.config.kanjicomponents = localStorage["kanjicomponents"];
-	chrome.extension.getBackgroundPage().rcxMain.config.kanjiinfo = kanjiinfoarray;
-	chrome.extension.getBackgroundPage().rcxMain.config.lineEnding = localStorage["lineEnding"];
-	chrome.extension.getBackgroundPage().rcxMain.config.copySeparator = localStorage["copySeparator"];
-	chrome.extension.getBackgroundPage().rcxMain.config.maxClipCopyEntries = localStorage["maxClipCopyEntries"];
-	chrome.extension.getBackgroundPage().rcxMain.config.popupLocation = localStorage["popupLocation"];
-}
-window.onload = fillVals;
-
-/*function clicktab(tab) {
-	selectedtab = document.getElementById(tab);
-	// change format of all tabs to deselected
-	// change format of selected tab to selected
-	// hide all tab contents
-	// show selected tab contents
-}*/
-
-
-document.querySelector('#submit').addEventListener('click', getVals);
-
+
+//Todo: Update config
+//TOdo: add the second tobi addition
+// Get the current options.
+function get_options() {
```

Comment:
JS style is camel case like `getOptions` but in this case since this doesn't return a value maybe it would be better like:

`initializeFormFromCloudSync` or similar.